dxtbx/format/nexus.py

```diff
@@ -1186,8 +1186,8 @@ def set_frame(pg, transformation):
             for module_number, nx_detector_module in enumerate(modules):
                 # Get the depends_on starting point for this module and for this image
                 panel_name = str(os.path.basename(nx_detector_module.handle.name))
-                px_fast = int(nx_detector_module.handle["data_size"][0])
-                px_slow = int(nx_detector_module.handle["data_size"][1])
+                px_fast = int(nx_detector_module.handle["data_size"][-1])
```

Comment:
Below we store the size as a tuple and use `map()` - would be cleaner to do the same here?